packages/viewer/src/lib/interactions/interactionApi.ts

```diff
@@ -506,6 +506,42 @@ export abstract class InteractionApi {
     });
   }
 
+  /**
+   * Performs a pivot operation of the scene's camera, updating the lookAt
+   * while maintaining the position, and requests a new image for the
+   * updated scene.
+   *
+   * @param degreesLocalX The angle to rotate the lookAt point around the local x-axis
+   * @param degreesLocalY The angle to rotate the lookAt point around the local y-axis
+   */
+  public async pivotCamera(
+    degreesLocalX: number,
+    degreesLocalY: number
+  ): Promise<void> {
+    return this.transformCamera(({ camera }) => {
+      const { position, up, lookAt } = camera;
+      const normalizedUp = Vector3.normalize(up);
+      const normalizedViewVector = Vector3.normalize(camera.viewVector);
+      const xVector = Vector3.cross(normalizedUp, normalizedViewVector);
+      const yVector = Vector3.cross(normalizedViewVector, xVector);
+
+      const updatedLookAtX = Vector3.rotateAboutAxis(
+        Angle.toRadians(degreesLocalX),
+        lookAt,
+        xVector,
+        position
+      );
+      const updatedLookAtY = Vector3.rotateAboutAxis(
+        Angle.toRadians(-degreesLocalY),
```

Comment:
Why is degreesLocalY negative?

Reply:
This is related to the way the mouse movement deltas are computed, (top -> bottom is positive vertically). That being said, this was pulled from logic in mfg-ui, and could probably stand to be updated and just have the interaction itself handle that. I'll take a look at updating that